src/Set/Packages/Livewire/LivewireLevelSetList.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RectorLaravel\Set\Packages\Livewire;
+
+use Rector\Set\Contract\SetListInterface;
+
+final class CashierLevelSetList implements SetListInterface
```

Comment:
I think you missed rename here 🤔

Reply:
Thank you @MrPunyapal , I've fixed that now